acceptance_tests/test_wfs.py

```diff
@@ -0,0 +1,7 @@
+from xml.etree import ElementTree
+
+
+def test_get_feature(connection):
+    answer = connection.get_xml('?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAME=poi&featureId=polygons.foo')
```

Comment:
I am wondering if here the base url is http://localhost:8080/ or so instead of localhost:8080/geoserver/tiger/ows?